blog/content/post/2020-08-06-race-condition-debugging.md

```diff
@@ -0,0 +1,210 @@
+---
+title: Case study - debugging race condition
```

Comment:
@IgorBagnucki Maybe add `in golang`? The title probably should inform which language/technology is used